web/src/components/login-form-google.js

```diff
@@ -75,11 +75,12 @@ class LoginFormGoogle extends Component {
                         this.changeIsMember(
                             /* isSucess= */ data.signInWithGoogle.success,
                             /* token= */ data.signInWithGoogle.token);
-                        this.redirect();
+                        // this.redirect();
```

Comment:
If you want to remove this line (not sure why), let's just delete this instead of just commenting out.

Reply:
I got it! I remove it because I put the redirect inside the changeIsMember function!